src/helper/task.ts

```diff
@@ -735,6 +737,20 @@ export class ZkWasmServiceHelper {
     return config;
   }
 
+  async queryProverNodeTimerangeStats(
+    query: ProverNodeTimerangeStatsParams,
+  ): Promise<ProverTimeRangeStats> {
+    const config = await this.endpoint.invokeRequest(
```

Comment:
it is the result of prover node's time range state so "config" better change to "result" at least, or "stats".